spec/factories/opportunities.rb

```diff
@@ -0,0 +1,12 @@
+require 'faker'
+
+FactoryBot.define do
+  factory :opportunity, class: OpenStax::Salesforce::Remote::Opportunity do
+    skip_create
+    id {Faker::Alphanumeric.alphanumeric(number: 10, min_alpha: 3, min_numeric: 3)}
+    term_year { [ '2019 - 20 Spring', '2019 - 20 Fall', '2018 - 19 Fall', '2018 - 19 Spring'].sample }
```

Comment:
will this have to be updated as the years progress or does it not matter?

Reply:
@mwharrison I'm not sure it matters. If they add summer, we should add that for completeness.